sdks/java/extensions/zetasketch/src/main/java/org/apache/beam/sdk/extensions/zetasketch/HllCount.java

```diff
@@ -107,6 +109,20 @@
   // Cannot be instantiated. This class is intended to be a namespace only.
   private HllCount() {}
 
+  /**
+   * Returns the sketch stored as bytes in the input {@code ByteBuffer}. If the input {@code
```

Comment:
Was confused here at first (when not being aware that this is a static method, one can read it as "the user passes in an empty/reusable ByteBuffer and the library returns the serialized sketch in that ByteBuffer"). 
To avoid that not-so-alert readers think that ;), how about 
"Converts the passed-in sketch from ByteBuffer to byte[], mapping null ByteBuffers (representing empty sketches) to empty byte[]. Utility method to convert sketches materialized with ZetaSQL/BigQuery to valid input for HllCount transforms."?

Reply:
Done. Agree that this does sounds more clear!